### PR TITLE
Add Post: dAppling Introduction

### PR DESCRIPTION
**Pull request by Namaskar-1F64F**
Howdy, ya'll. I crafted this post about my app: [dAppling](https://dappling.network). We build and deploy code to IPFS. This is a walkthrough of how a user can deploy a simple blog using the platform.

### CODE REVIEW

**Comment by Namaskar-1F64F** [comment]
> @Namaskar-1F64F Do you have an ETA of when your product will be out of beta?
I do not

**Comment by Namaskar-1F64F** [comment]
> We've got several blog posts lined up, so will probably not get this one published for another week or so

sounds good!

> one piece of feedback from the team was the reference to "100% up time" being a bit unrealistic when even Filebase and others can't guarantee such a thing. We'd recommend tweaking that a bit.

good feedback, and makes sense. the intention wasn't to say "we guarantee uptime" but more like, "if you use dappling in addition to your existing frontend, you can get much better up-time". Either way, I will get that changed.

File: src/_blog/2023-09-introducing-dappling.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+Basically, we take your code, build it, and host the resulting files on IPFS. But what makes us special are the features we provide to make your experience easier. Even if you have an existing site, you can use dAppling to create a resilient "alternative frontend" that is hosted on IPFS.
+
+# The Walkthrough
+
+I will guide you step by step through the process of deploying a project on dAppling. I will be using a blog template that I created, but you can use any code that you want to deploy.
+
+The final project, [on the dAppling platform](https://dappling.network/projects/7ebe4f4f-70f0-4705-828b-c610fb1d9ddc), will be accessible at a URL like [hello-ipfs-blog.dappling.network](https://hello-ipfs-blog.dappling.network).
+
+Let's start exploring!
```
Comment (lidel) [inline src/_blog/2023-09-introducing-dappling.md:37]: :warning:  I feel this entire tutorial section that follows does not work well on blog, and should live in https://docs.dappling.network instead. Could it be replaced with a short summary and a link to your docs? 

Perhaps https://docs.dappling.network/getting-started/? (or create dedicated single-page 101 tutorial there).
This way the blog post will not get old / outdated as your project evolves, and you don't overwhelm reader with wall of screenshots how to create a github repo :-)

**Review by lidel — DISMISSED** [review]
HI @Namaskar-1F64F, thank you for submititng this.

It is great to see projects which glue multiple dweb protocols in user-friendly product, would like to figure out a way to connect you with potential users.  
Given how extensive the 101 tutorial here is, and how many screenshots and hands-on things it includes, this blog may not be the best venue, or the content needs some adjustment.

Some thoughts, hope you find them helpful:


- :point_right:  Did you consider **publishing this 101 tutorial  on your own blog instead?** 
   - Focusing on this would be my suggestion, you really want to control the origin of your content.
   - This way you could get featured in  [IPFS Newsletter](https://ipfs.us4.list-manage.com/subscribe?u=25473244c7d18b897f5a1ff6b&id=cad54b2230) for increasing the reach.  The newsletter is sent to users via email and also gets [published to the blog](https://blog.ipfs.tech/?tags=weekly) and https://discuss.ipfs.tech/, so you effectively reuse content to reach more potential users.

- :warning:  If you wish to land a dedicated post on blog.ipfs.tech, the content might need to be revised:
  - The entire "The Walkthrough"  buries the lead, waters down your value proposition and hides features like ENS/IPNS/your "AI" integration behind mundane screenshots of "how to create a github repo" :) 
     - For an engaging blogpost, you may want to focus on condensed primer on the value your project brings, e.g. how it takes care of publishing CID using multiple protocols (ENS/DNS/IPNS) to maximize resiliency. 
     - The step-by-step guide feels like something that should live in as a tutorial in your own blog or https://docs.dappling.network/, after you hook reader's attention (see comment inline).
  - For baseline, see similar product showcases from the past (they aged relatively well):
    - https://blog.ipfs.tech/2022-04-24-fleek/
    - https://blog.ipfs.tech/2022-04-14-filebase/ 

ps. you could also:
- Consider adding your project to https://ecosystem.ipfs.tech/ and https://github.com/ipfs/awesome-ipfs/
- Add yourself to the list of services at http://docs.ipfs.tech/concepts/persistence/#pinning-services
- Submit a [case study](https://docs.ipfs.tech/case-studies/) (on your own, or joint with one of your clients)

File: src/_blog/2023-09-introducing-dappling.md (hunk 1 above)
Comment (Namaskar-1F64F) [reply src/_blog/2023-09-introducing-dappling.md:37]: I think your suggestions are a great idea. How are we able to get featured in the IPFS newsletter? Happy to host this 101 content on our blog + promote in the newsletter, then create a simpler / revised post for the IPFS blog.
Great tips on the ecosystem/docs/case study -> we will work on those.

Comment (RusseII) [reply src/_blog/2023-09-introducing-dappling.md:37]: Hi all. @Namaskar-1F64F 's cofounder here. Thanks for the great suggestions! 

@lidel just [submitted](https://cln.sh/rPHgnj2B) to the IPFS ecosystem page & already on the awesome-ipfs repo 😄

Comment (Namaskar-1F64F) [reply src/_blog/2023-09-introducing-dappling.md]: thanks @damedoteth I sent in a request 🌌

**Comment by Namaskar-1F64F** [comment]
I've updated the blog to be more of a "here's what we do" sort of article. sharing key features of ease of use + access.

File: src/_blog/2023-10-introducing-dappling.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+#### Points of Access
+
+There are a couple of ways to access your site. When the code is built and uploaded to IPFS, you will receive what is called a [Content Identifier (CID)](https://docs.ipfs.tech/concepts/content-addressing/). It's basically the hash of all your files.
+
+You will receive a new CID every time your site is re-built because the resulting files have changed. Luckily, we use the [InterPlanetary Name System (IPNS)](https://docs.ipfs.tech/concepts/ipns/) to create a key that will always point to the most recent CID.
+
+So the most straightforward way to fetch your content would be directly from an [IPFS node](https://docs.ipfs.tech/concepts/nodes/). Since not everyone is running an IPFS node (yet), you can instead use an [IPFS gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/) in which a third party fetches the content from their node and serves it over HTTPS.
+
+Since we store the on our `dappling.eth` ENS name, you can also fetch the content through a service like [eth.limo](https://eth.limo). This service first reads the IPNS key that we set, resolves it to a CID, and then serves the content like a gateway.
```
[the remark this answers is not in the document]
Comment (Namaskar-1F64F) [reply src/_blog/2023-10-introducing-dappling.md]: updated the text here. it's the IPNS key that's being stored as the content hash, but didn't want to get that technical

**Comment by Namaskar-1F64F** [comment]
thanks for the suggestions. they all seem sensible.

**Comment by Namaskar-1F64F** [comment]
Closing in favor of #657 🙏

**Pull request closed without merging**